scripts/module/stadium.nut

```diff
@@ -0,0 +1,25 @@
+// Stadium billboards
+
+currentModelOffset <- 0;
+BillboardsBegin <- 2042;
+
+function onScriptLoad() {
+  billboard <- CreateObject(BillboardsBegin, 1, -1037.08, 1340.258, 36.552, 255);
+}
+
+function onPlayerCommand( player, cmd, text ) {
+  if (! player.Admin ) {
+    return;
+  }
```

Comment:
I'd recommend moving it into the billboard step itself (so we can add other user-function), OR, moving the entire function into the admin function-set (so it's a core function like setting time).

Reply:
This is how it's done in other scripts iirc with the exception that other scripts actually has user scripts in place before the admin check.

Reply:
Fair enough.